data/names/ru.json

```diff
@@ -5870,7 +5870,624 @@
+      [ "Герра", "Guerra" ],
+      [ "Гиллиленд", "Gilliland" ],
+      [ "Чейз", "Chase" ],
+      [ "Ховард", "Howard" ],
+      [ "Гойнс", "Goins" ],
+      [ "Хэд", "Head" ],
+      [ "Гольдберг", "Goldberg" ],
+      [ "Гонсалвес", "Gonsalves" ],
+      [ "Кук", "Cook" ],
+      [ "Граббс", "Grubbs" ],
+      [ "Грейвс", "Graves" ],
+      [ "Грейнджер", "Granger" ],
+      [ "Гриер", "Grier" ],
+      [ "Гриффин", "Griffin" ],
+      [ "Гроувер", "Grover" ],
+      [ "Грэм", "Graham" ],
+      [ "Гуахардо", "Guajardo" ],
+      [ "Гузман", "Guzman" ],
+      [ "Гундерсон", "Gunderson" ],
+      [ "Гэлвин", "Galvin" ],
+      [ "Давенпорт", "Davenport" ],
+      [ "Дарлинг", "Darling" ],
+      [ "Дайкс", "Dykes" ],
+      [ "Данжело", "Dangelo" ],
+      [ "Данэм", "Dunham" ],
+      [ "Дарби", "Darby" ],
+      [ "Дасилва", "Dasilva" ],
+      [ "Дауди", "Dowdy" ],
+      [ "Ноублз", "Nobles" ],
+      [ "Деврис", "Devries" ],
+      [ "Дехесус", "Dejesus" ],
+      [ "Делакрус", "Delacruz" ],
+      [ "Делароза", "Delarosa" ],
+      [ "Демерс", "Demers" ],
+      [ "Дэй", "Day" ],
+      [ "Вуд", "Wood" ],
+      [ "Джайлз", "Giles" ],
+      [ "Джейкобсон", "Jacobson" ],
+      [ "Джеймисон", "Jamison" ],
+      [ "Джентайл", "Gentile" ],
+      [ "Джентри", "Gentry" ],
+      [ "Джии", "Gee" ],
+      [ "Джилетт", "Gillette" ],
+      [ "Джилл", "Gill" ],
+      [ "Джуэлл", "Jewell" ],
+      [ "Джёрман", "German" ],
+      [ "Кауч", "Couch" ],
+      [ "Диггс", "Diggs" ],
+      [ "Диль", "Diehl" ],
+      [ "Дион", "Dion" ],
+      [ "Лонг", "Long" ],
+      [ "Дов", "Dove" ],
+      [ "Дозье", "Dozier" ],
+      [ "Дейл", "Dale" ],
+      [ "Хаус", "House" ],
+      [ "Доти", "Doty" ],
+      [ "Драйвер", "Driver" ],
+      [ "Дрейк", "Drake" ],
+      [ "Дрейпер", "Draper" ],
+      [ "Френд", "Friend" ],
+      [ "Дуайер", "Dwyer" ],
+      [ "Дюбе", "Dube" ],
+      [ "Дусетт", "Doucette" ],
+      [ "Дэйвид", "David" ],
+      [ "Дэйгл", "Daigle" ],
+      [ "Дэйли", "Dailey" ],
+      [ "Дюбоз", "Dubose" ],
+      [ "Дюк", "Duke" ],
+      [ "Дюкс", "Dukes" ],
+      [ "Дюпри", "Dupree" ],
+      [ "Бишоп", "Bishop" ],
+      [ "Жан", "Jean" ],
+      [ "Вест", "Vest" ],
+      [ "Лайт", "Light" ],
+      [ "Лоу", "Law" ],
+      [ "Лоз", "Laws" ],
+      [ "Сток", "Stock" ],
+      [ "Зейглер", "Zeigler" ],
+      [ "Зейтц", "Seitz" ],
+      [ "Зепеда", "Zepeda" ],
+      [ "Винтер", "Winter" ],
+      [ "Винтерс", "Winters" ],
+      [ "Голд", "Gold" ],
+      [ "Ингланд", "England" ],
+      [ "Иризарри", "Irizarry" ],
+      [ "Айрленд", "Ireland" ],
+      [ "Спаркс", "Sparks" ],
+      [ "Йе", "Yee" ],
+      [ "Йейтс", "Yates" ],
+      [ "Кавасос", "Cavazos" ],
+      [ "Казарес", "Cazares" ],
+      [ "Хай", "High" ],
+      [ "Стоун", "Stone" ],
+      [ "Рид", "Reed" ],
+      [ "Карбахал", "Carbajal" ],
+      [ "Карвалью", "Carvalho" ],
+      [ "Кардуэлл", "Cardwell" ],
+      [ "Карриер", "Carrier" ],
```

Comment:
Supposed to be `Кэрриер`. The book *A Dictionary of English Surnames* also provides `Каррир`, which may be one of those transcriptions that rely on different pronunciations of the same name; such as in case of Houston (`Хьюстон`, like the city in Texas, and `Хаустон`, like the person after whom the street in New York City is named).

I suggest `Кэрриер`, since that's what one would expect to read it as.